scripts/train_catsdogs.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-python train.py --epochs 250 --optimizer Adam --lr 0.001 --wd 0 --deterministic --compress policies/schedule-catsdogs.yaml --model ai85cdnet --dataset cats_vs_dogs --confusion --param-hist --embedding --device MAX78000 "$@"
+python train.py --epochs 200 --optimizer Adam --lr 0.001 --wd 0 --deterministic --compress policies/schedule-catsdogs.yaml --qat-policy policies/qat_policy_late_cd.yaml --model ai85cdnet --dataset cats_vs_dogs --confusion --param-hist --embedding --device MAX78000 "$@"
```

Comment:
Seems that "qat_policy_late_cd.yaml" should be replaced with "qat_policy_cd.yaml".

Reply:
Correct and fixed...